cmd/av/stack_sync.go

```diff
@@ -309,4 +310,29 @@ func init() {
 	stackSyncCmd.MarkFlagsMutuallyExclusive("current", "trunk")
 	stackSyncCmd.MarkFlagsMutuallyExclusive("trunk", "parent")
 	stackSyncCmd.MarkFlagsMutuallyExclusive("continue", "abort", "skip")
+
+	if err := stackSyncCmd.RegisterFlagCompletionFunc("parent", func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+		return allBranches(), cobra.ShellCompDirectiveDefault
+	}); err != nil {
+		log.Fatal(err)
+	}
+}
+
+func allBranches() []string {
+	repo, err := getRepo()
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Return nil on error. For auto completion candidates, I think we don't need to (or be able to show) errors.